### PR TITLE
Fix bug with separation_3d failing when velocity data present

### PR DESCRIPTION
**Pull request by adrn**
In master:
```python
c1 = SkyCoord(ra=138*u.deg, dec=-17*u.deg, distance=100*u.pc,
             pm_ra_cosdec=5*u.mas/u.yr,
             pm_dec=-7*u.mas/u.yr,
             radial_velocity=160*u.km/u.s)
c2 = SkyCoord(ra=138*u.deg, dec=-17*u.deg, distance=105*u.pc,
             pm_ra_cosdec=15*u.mas/u.yr,
             pm_dec=-74*u.mas/u.yr,
             radial_velocity=-60*u.km/u.s)

sep = c1.separation_3d(c2)
```
results in
```
ERROR: TypeError: Operation 'sub' is not supported when differentials are attached to a CartesianRepresentation.
```

But computing the positional separation between two coordinates shouldn't care about velocities! 

Note @bsipocz - it would be good to have this in 3.0.2, and out ASAP because I suspect this will be an issue for Gaia data users (I found the bug when playing with some mock DR2 data 😄)

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -277,6 +277,10 @@ astropy.convolution
 astropy.coordinates
 ^^^^^^^^^^^^^^^^^^^
 
+- Computing a 3D separation between two ``SkyCoord`` objects (with the
+  ``separation_3d`` method) now works with or without velocity data attached to
+  the objects. [#9999]
```
Comment (pllim) [inline CHANGES.rst]: Need to replace "9999" with the correct PR number.

File: astropy/coordinates/tests/test_sky_coord_velocities.py [hunk 2]
```diff
@@ -200,3 +200,16 @@ def test_position_angle(sc, sc_for_sep):
 def test_constellations(sc):
     const = sc.get_constellation()
     assert const == 'Pisces'
+
```
Comment (pllim) [inline astropy/coordinates/tests/test_sky_coord_velocities.py]: PEP8 nitpick: 2 blank lines between functions.

**Comment by adrn** [comment]
And hope no one slips in a new issue/PR in the intervening time :) -- thanks for the link, that is exactly what I need!

**Comment by pllim** [comment]
The green status of CI is misleading. There is an error in the second commit. Is it related? I don't see that error on `master`.

https://travis-ci.org/astropy/astropy/builds/368135956

**Comment by pllim** [comment]
All clear. Merging!

**Comment by eteq** [comment]
👍 (although after the fact...)

**Pull request merged**